### DISCUSSION TITLE
Using Filters to retrieve TOP n rows

### DISCUSSION THREAD

**Discussion by needledrag** [Mimetis/Dotmim.Sync#424] [Q&A] [answered]
Hi all, 

Was wondering if anyone has been able to figure out how to restrict server row selection to a specific number of rows eg TOP 100 in a filter?

Also, and I may have missed it in the docs, is there anyway for comparison operators with a filter eg less than or greater than ?  Or does this have to be restricted on the selected rows in the response?

Regards
Shaun

**Comment by Mimetis** [comment, marked as answer]
You should use `CustomWhere` I guess
